apps/whale-api/src/module.api/poolswap.pathfinding.service.ts

```diff
@@ -23,15 +22,21 @@ export class PoolSwapPathFindingService {
   tokensToSwappableTokens = new Map<TokenIdentifier['id'], TokenIdentifier[]>()
 
   constructor (
-    protected readonly poolPairTokenMapper: PoolPairTokenMapper,
     protected readonly deFiDCache: DeFiDCache,
     @Inject('NETWORK') protected readonly network: NetworkName
   ) {
   }
 
   @Interval(120_000) // 120s
   async syncTokenGraph (): Promise<void> {
-    const poolPairTokens = await this.poolPairTokenMapper.list(200)
+    const poolPairsResult = await this.deFiDCache.listPoolPairs(60) as PoolPairsResult
```

Comment:
`question:` if the method being called can potentially return `undefined`, should this be accounted for in this method by doing a defensive check?

Providing a `undefined` value to `Object.entries` would produce an error in this method.